bindings_ffi/src/mls.rs

```diff
@@ -8039,19 +7767,12 @@ mod tests {
     #[tokio::test]
     async fn test_sync_consent() {
         // Create two test users
-        let wallet_alix = generate_local_wallet();
-        let wallet_bo = generate_local_wallet();
-
-        let alix =
-            new_test_client_with_wallet_and_history_sync_url(wallet_alix.clone(), None, None).await;
-        let alix_worker = alix.inner_client.worker_handle().unwrap();
-        alix_worker.wait_for_init().await.unwrap();
-        let bo = new_test_client_with_wallet_and_history_sync_url(
-            wallet_bo,
-            None,
-            Some(FfiSyncWorkerMode::Disabled),
-        )
-        .await;
+        let alix = Tester::builder()
```

Comment:
Tester now has a builder. Want to eventually get rid of those half-dozen functions with names like `new_test_client_with_wallet_and_history_sync_url`